src/App/Resources/AppResources.it.resx

```diff
@@ -2613,7 +2613,7 @@ Vuoi passare a questo account?</value>
     <value>US</value>
   </data>
   <data name="EU" xml:space="preserve">
-    <value>UE</value>
+    <value>EU</value>
```

Comment:
🤔  I believe that if we want to do any change on resources we need to go to Crowdin and add a translation over there to be then impacted on the code. Is this correct @djsmith85 or can we also apply direct changes on translations without going through Crowdin?